apps/emqx_oracle/src/emqx_oracle_schema.erl

```diff
@@ -38,3 +38,19 @@ service_name(type) -> binary();
 service_name(desc) -> ?DESC(?REF_MODULE, "service_name");
 service_name(required) -> false;
 service_name(_) -> undefined.
+
+adjust_fields(Fields) ->
+    lists:map(
+        fun
+            ({username, OrigUsernameFn}) ->
+                {username, fun
+                    (required) ->
+                        true;
```

Comment:
Note: this breaks backward compatibility (EMQX won't start if it has an Oracle bridge config without username).   
Anyway, Oracle bridges with no user seem to be useless (can't be connected), and we did it previosuly for other bridges, e.g. PostgreSQL: https://github.com/emqx/emqx/commit/8cee75139d